cardinal/world/vars.go

```diff
@@ -0,0 +1,12 @@
+package world
+
+type WorldVarsKey string
+
+const (
+	PlayerCount = WorldVarsKey("playerCount")
```

Comment:
this `PlayersCount` is only used in this file, so the addition of the `WorldVarsKey` type seems unnecessary.

was this key supposed to be used somewhere else in the code?

Reply:
yeah, will fix that.